app/src/sharedTest/java/org/oppia/android/app/parser/CustomBulletSpanTest.kt

```diff
@@ -87,43 +89,60 @@ class CustomBulletSpanTest {
   @Test
   fun customBulletSpan_testReplaceBulletSpan_spannableStringWithoutBulletSpanRemainSame() {
     val spannableString1 = testStringWithoutBulletSpan
-    val spannableString2 = CustomBulletSpan.replaceBulletSpan(spannableString1, context)
-    assertThat(getNoofBulletSpans(spannableString1)).isEqualTo(0)
-    assertThat(getNoofCustomBulletSpans(spannableString1)).isEqualTo(0)
-    assertThat(getNoofBulletSpans(spannableString2)).isEqualTo(0)
-    assertThat(getNoofCustomBulletSpans(spannableString2)).isEqualTo(0)
+    val convertedSpannableStringBuilder = CustomBulletSpan.replaceBulletSpan(
+      SpannableStringBuilder(spannableString1),
+      context
+    )
+    val spannableString2 = SpannableString.valueOf(convertedSpannableStringBuilder)
```

Comment:
Can we instead use ``convertedSpannableStringBuilder.build()``?

Reply:
@BenHenning No, there is no property `build()` for `SpannableStringBuilder`

Reply:
Confirmed that the suggested change is not possible.